src/position.ts

```diff
@@ -459,7 +456,7 @@ export class UserPosition extends PositionWithRunner {
 
       case 'wstETH':
         gasEstimate = await this.positionManager.managePosition.estimateGas(
-          RaftConfig.getTokenAddress(collateralToken),
+          RaftConfig.getTokenAddress(collateralToken) as string,
```

Comment:
Same here, throw error if token is null.

Reply:
Should never happen since `'wstETH'` has a token address.